### PR TITLE
Dev/jgpoland/io 4532 api docs housekeeping

### PR DESCRIPTION
**Pull request by GimmeYerGold**
CHANGES

• removed 8080 dev comment
• fixed errors addressed by api markdown linter
• revised some response and request examples

### CODE REVIEW

File: api/endpoints/move.md [hunk 1]
```diff
@@ -626,11 +629,13 @@ To update a Move Request send JSON with updated values for one or more of the at
...
 
            {
+             {
                 "response": "Successfully removed"
+             }
            }
 
 
-## Move Collection [/move/requests///{{?statusId,statusCode}}]
+## Move Collection [/move/requests/{?statusId,statusCode}]
```
Comment (rmoscowitz) [inline api/endpoints/move.md]: Does this guy still work with only one `/`?

**Review by rmoscowitz — COMMENTED** [review]
Looks good! Only one comment.

**Comment by GimmeYerGold** [comment]
@rmoscowitz i addressed your comment by escaping *all* curly braces like so `///{thingy}`

I also found a lot of little other things that needed tweaking 😄

**Review by rmoscowitz — APPROVED** [review]
LGTM!

**Pull request merged**